DataFormats/PatCandidates/interface/PackedCandidate.h

```diff
@@ -716,6 +724,8 @@ namespace pat {
         lostInnerHitsMask = 0x30, lostInnerHitsShift=4,
         muonFlagsMask = 0x0600, muonFlagsShift=9
     };
+  public:
+    uint16_t firstHit_;
```

Comment:
it looks like I missed that this was added as a public member in the initial review.
@cms-sw/core-l2 is it safe to change this to private or would the schema evolution/root have problems with bw compatibility?